### ISSUE TITLE
Validate does not allow Alternate Title for context products

### ISSUE THREAD

**Issue by matthewtiscareno** [NASA-PDS-Incubator/validate#1122]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

We are running Validate on a bundle.  There are many examples in which we cite a context product using an Alternate Title, and Validate throws a warning.  For example:
```
WARNING  [warning.label.context_ref_mismatch]   line 178: Context reference name mismatch. Value: 'Alpha Crucis' Expected one of: '[Acrux]'
```

This warning comes from the following label excerpt:
```
<Target_Identification>
    <name>Alpha Crucis</name>
    <alternate_designation>Alpha Cru</alternate_designation>
    <type>Star</type>
    <Internal_Reference>
        <lid_reference>urn:nasa:pds:context:target:star.alf_cru</lid_reference>
        <reference_type>data_to_target</reference_type>
    </Internal_Reference>
</Target_Identification>
```

We are seeing many similar examples.  These warnings should not appear.  It should not be wrong to use the Alternate Title as the Title. 

Looking at [the context product being referenced](https://pds.nasa.gov/data/pds4/context-pds4/target/star.alf_cru_1.0.xml), the Bayer designation (Greek letter and constellation) is an `Identification_Area.Alias_List.Alias.Alternate Title`, but the common name being requested is both `Identification_Area.title` and `Target.name`.  

### 🕵️ Expected behavior

I expected Validate to not throw a warning when `Target_Identification.name` in a product label matches `Identification_Area.Alias_List.Alias.Alternate Title` (rather than `Target.name`) in the referenced context product.

### 📚 Version of Software Used

Validate 3.6.3

### 🦄 Related requirements

🦄 #xyz

### Acceptance Criteria

**Given** 
**When I perform** 
**Then I expect**

**Comment by jordanpadams** [comment]
@matthewtiscareno we have a ticket for a new requirement to support aliases here:  https://github.com/NASA-PDS/validate/issues/970 . I will close this out as a duplicate, and we can look to address that in a future build if the SWG agrees with this.
